packages/transport/test/end2end/End2End.test.ts

```diff
@@ -420,6 +420,51 @@ describe("RelationshipTest: Revoke", function () {
     });
 });
 
+describe("RelationshipTest: Terminate", function () {
+    let connection: IDatabaseConnection;
+    let transport: Transport;
+
+    let from: AccountController;
+    let to: AccountController;
+
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+        transport = TestUtil.createTransport(connection);
+
+        await transport.init();
+
+        const accounts = await TestUtil.provideAccounts(transport, 2);
+        from = accounts[0];
+        to = accounts[1];
+    });
+
+    afterAll(async function () {
+        await from.close();
+        await to.close();
+        await connection.close();
+    });
+
+    test("should terminate a relationship between two accounts", async function () {
+        const relationshipId = (await TestUtil.addRelationship(from, to)).acceptedRelationshipFromSelf.id;
+
+        const terminatedRelationshipFromSelf = await from.relationships.terminate(relationshipId);
+        expect(terminatedRelationshipFromSelf.id.toString()).toStrictEqual(relationshipId.toString());
+        expect(terminatedRelationshipFromSelf.status).toStrictEqual(RelationshipStatus.Terminated);
+        expect(terminatedRelationshipFromSelf.cache?.auditLog).toHaveLength(3);
+        expect(terminatedRelationshipFromSelf.cache!.auditLog[2].newStatus).toBe(RelationshipStatus.Terminated);
+
+        // Get terminated relationship
```

Comment:
Can this be omitted?

Reply:
I've decided to follow the existing style

Reply:
I don't mind those comments, I like them for structure, but it seems I'm outvoted. Now I'm curious about your opinion on the SyncController tests.